ml-agents/mlagents/trainers/tensorflow_to_barracuda.py

```diff
@@ -1499,23 +1499,13 @@ def very_slow_but_stable_topological_sort(nodes, verbose):
 #########################################################
```

Comment:
This file is created by the barracuda team, you might want to tell them what you are doing so they can incorporate it in their version rather than having to maintain 2 different versions of the same feature.

Reply:
I've reached out to @mantasp to figure out how best to contribute.